{
  final Exception[] thrown=new Exception[1];
  UIUtil.invokeAndWaitIfNeeded(new Runnable(){
    @Override public void run(){
      try {
        DataMPSFixtureTestCase.super.setUp();
      }
 catch (      Exception e) {
        thrown[0]=e;
      }
    }
  }
);
  if (thrown[0] != null)   throw thrown[0];
}
